keel-core/src/main/kotlin/com/netflix/spinnaker/keel/exceptions/DuplicateArtifactReferenceException.kt

```diff
@@ -0,0 +1,8 @@
+package com.netflix.spinnaker.keel.exceptions
+
+class DuplicateArtifactReferenceException(
```

Comment:
As I mentioned in the other comment, if you add this exception (or perhaps the parent `ValidationException`) to the handler below, then it will cause the API to return a 400:
https://github.com/spinnaker/keel/blob/4c5a4852401eee558709ac987703ab2e178ad771/keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ExceptionHandler.kt#L40-L42

Reply:
Addressed in 8f805d44.